src/objects/address.ts

```diff
@@ -0,0 +1,27 @@
+import { ICityPostCode } from '../models/address';
+import * as fs from "fs";
+import * as path from "path";
+
+const cities: ICityPostCode[] = [];
+const filePath = path.resolve(__dirname, '../env/belgian-cities.csv');
+const data = fs.readFileSync(filePath, { encoding: 'utf-8' }).split('\n');
+
+function parseCities() {
+    
+    for(let i = 0; i < data.length; i++)
+    {
+        const row = data[i].split(',');
+        const tempCity: ICityPostCode = {
+            postal_code: row[0],
+            city_name: row[1]
+        }
+        cities.push(tempCity);
+    }
+}
```

Comment:
This parser might cause issues with splitting (for example line 390). Likely better to use a library for this. `csv-parse` is a library you can use. Alternatively load the json file instead.

Reply:
Also my mistake, but why is this in ad-data and not in the SQL scripts?
I would assume this goes along with init sql script. We currently only have GET routes planned for the city routes.
Maybe there is an option for postgresql to insert csv with a script.

Reply:
> Also my mistake, but why is this in ad-data and not in the SQL scripts? I would assume this goes along with init sql script. We currently only have GET routes planned for the city routes. Maybe there is an option for postgresql to insert csv with a script.

There is
```
COPY persons(first_name, last_name, dob, email)
FROM 'C:\sampledb\persons.csv'
DELIMITER ','
CSV HEADER;
```
source: https://www.postgresqltutorial.com/postgresql-tutorial/import-csv-file-into-posgresql-table/